packages/suite-data/files/translations/af.json

```diff
@@ -1,7 +1,7 @@
 {
   "AMOUNT": "Bedrag",
-  "AMOUNT_IS_LESS_THAN_RESERVE": "Recipient account requires minimum reserve {reserve} XRP to activate\nOntvanger vereis minstens {reserve} XRP reserwe om te aktiveer",
-  "AMOUNT_IS_MORE_THAN_RESERVE": "Bedrag is meer as die vereiste on-spandeerbare reserwe  ({reserve} XRP)",
+  "AMOUNT_IS_LESS_THAN_RESERVE": "Recipient account requires minimum reserve {reserve} {symbol} to activate\nOntvanger vereis minstens {reserve} {symbol} reserwe om te aktiveer",
```

Comment:
please discard all translation changes. Keep only `messages.json` + `en.json`

Reply:
This issue was addressed during the rebase.